mm2src/mm2_net/src/transport.rs

```diff
@@ -76,7 +76,8 @@ pub struct GuiAuthValidationGenerator {
     pub address: String,
 }
 
-/// gui-auth specific data-type that needed in order to perform gui-auth calls
+/// Gui-auth specific data-type that needed in order to perform gui-auth calls.
+/// Represents a signed message used for authenticating and validating requests processed by the proxy.
 #[derive(Clone, Serialize)]
 pub struct GuiAuthValidation {
```

Comment:
```suggestion
/// komodo-defi-proxy specific data types including values that are being validated (e.g., verifying signed messages and request expirations) by the komodo-defi-proxy server.
#[derive(Clone, Serialize)]
pub struct KomodoDefiProxyPayload {
```

Reply:
changed the names of different types here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2100/commits/36eaf389f1604136efefc3c4aabb159728dee1ac

I renamed `GuiAuthValidation` to `KomodefiProxyAuthValidation` because the payload types were previously named `AuthPayload` (now `QuicknodePayload`) and `HttpGetPayload` (now `MoralisPayload`).

new payload names are the same as here
https://github.com/KomodoPlatform/komodo-defi-proxy/commit/1c600306d86c8fcb1f91760c165be5a6f3b081fd